platform/include/platform.h

```diff
@@ -82,5 +82,8 @@ void platform_init(void);
 /* called by the arch init code to get the platform to set up any mmu mappings it may need */
 void platform_init_mmu_mappings(void);
 
+/* Called by LK to get the device tree */
+const void *get_fdt(void);
```

Comment:
Can you add a default weak reference version of this in platform/init.c (see the other __WEAK) functions, so that there's a default version that returns a null?

Reply:
Done